package.json

```diff
@@ -40,44 +40,46 @@
     "prepublishOnly": "npm run build",
     "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "wcag": "test-storybook"
   },
   "devDependencies": {
     "@babel/core": "^7.17.5",
     "@babel/preset-env": "^7.14.7",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.14.5",
-    "@percy/cli": "^1.0.7",
-    "@percy/storybook": "^4.2.0",
-    "@storybook/addon-a11y": "^6.4.20",
-    "@storybook/addon-actions": "^6.4.19",
-    "@storybook/addon-essentials": "^6.4.19",
-    "@storybook/addon-interactions": "^6.4.19",
-    "@storybook/addon-links": "^6.4.19",
+    "@percy/cli": "^1.7.2",
```

Comment:
wow this is a big bump 👀 
I checked out of curiosity for what changed and it looks like 1.8.0 was released 18 hours ago?

Reply:
upgraded percy/cli and storybook because it looks like they just released 6.5.10 yesterday

Reply:
oh I like to use https://www.npmjs.com/package/npm-check-updates to see what the latest versions for things in my package are, if you aren't using it already (it only reads from the package.json and not your package-lock)

Reply:
I didn't use it earlier for percy/cli though that was just me going to their github page